# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gcc/no-libvtv.patch.cross0
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-5.1.0/libvtv/configure.tgt.vanilla	2015-01-29 08:03:56.000000000 +0000
+++ gcc-5.1.0/libvtv/configure.tgt	2015-04-24 10:25:07.980000000 +0000
@@ -43,3 +43,4 @@
   *)
 	;;
 esac
+VTV_SUPPORTED=no
